### ISSUE TITLE
Potentially unnecessary tokenizer download on package init

### ISSUE THREAD

**Issue by ianbulovic** [Machine-Learning-for-Medical-Language/cnlp_transformers#213]
In [\_\_init\_\_.py](https://github.com/Machine-Learning-for-Medical-Language/cnlp_transformers/blob/main/src/cnlpt/__init__.py) we are downloading the "punkt" tokenizer on package initialization. As far as I can tell, punkt is not used anywhere in the package. We use the similarly named `wordpunct_tokenize`, but that is a different regex-based tokenizer. Removing the silent download should significantly speed up package import time.

I wanted to open this as an issue before submitting a PR, just in case we're using punkt somewhere that I missed.

**Comment by tmills** [comment]
Yeah, we've used punkt for splitting sentences before but we mostly use RuSH now (though we should probably reconsider with the trouble it causes with dependency management). I'm not sure why it was ever added to be honest.

**Comment by tmills** [comment]
Let's remove it and add it back if we figure out where/whether it's needed.
